benchmarks/TFLite/CMakeLists.txt

```diff
@@ -548,7 +548,6 @@ iree_benchmark_suite(
     "GPU-Adreno"
   TRANSLATION_FLAGS
     ${ANDROID_ADRENO_GPU_TRANSLATION_FLAGS}
```

Comment:
Whoo! This means no experimental flags here and in a few places below. With our current setup, I think that means we're going to be running the same benchmarks twice now :-/ We could drop the experimental flags benchmarks entirely, although that means we're not going to have history for them if/when we want to revive them. This is another reason to decouple benchmark data collection from the plotting of the data series. @antiagainst thoughts?